### PR TITLE
Fixes by Static analysis

### PR DESCRIPTION
**Pull request by szepeviktor**
...inspired by the current newsletter

All Discovered by @phpstan
Please consider running it in CI

@asisayag2 Feel free to ask questions!

### CODE REVIEW

**Comment by asisayag2** [comment]
Thanks @szepeviktor , 
We will take a look

File: cloudinary-image-management-and-manipulation-in-the-cloud-cdn/instance.php [hunk 1]
```diff
@@ -7,25 +7,23 @@
...
-global $cloudinary_plugin;
-
-if ( ! defined( 'DEBUG_SCRIPTS' ) ) {
-	define( 'CLDN_ASSET_DEBUG', '.min' );
-} else {
-	define( 'CLDN_ASSET_DEBUG', '' );
-}
+define( 'CLDN_ASSET_DEBUG', defined( 'DEBUG_SCRIPTS' ) ? '' : '.min' );
 
+require_once ABSPATH . 'wp-admin/includes/plugin.php';
```
Comment (szepeviktor) [inline cloudinary-image-management-and-manipulation-in-the-cloud-cdn/instance.php:12]: You have to comment out this line to run static analysis.

Actually `get_plugin_data()` reads **hard-coded** data from `cloudinary.php` :)

**Comment by DavidCramer** [comment]
Can't include to maintain WordPress VIP standards.

**Comment by szepeviktor** [comment]
@DavidCramer This is sad.

At least please grab my fixes.
Thanks.

**Comment by DavidCramer** [comment]
@szepeviktor no doubt the fixes are good. I will indeed pick from then.
If you would like, could you run this through WordPress VIP and clear those, then do another PR? That would help alot!

**Comment by szepeviktor** [comment]
Okay.
I hope you mean https://github.com/Automattic/VIP-Coding-Standards

**Pull request closed without merging**